druid-shell/src/platform/windows/window.rs

```diff
@@ -1075,8 +1079,20 @@ impl WindowHandle {
     // TODO: Implement this
     pub fn show_titlebar(&self, _show_titlebar: bool) {}
 
-    // TODO: Implement this
-    pub fn resizable(&self, _resizable: bool) {}
+    pub fn resizable(&self, resizable: bool) {
+        if let Some(w) = self.state.upgrade() {
+            let hwnd = w.hwnd.get();
+            unsafe {
+                let mut style = GetWindowLongPtrW(hwnd, GWL_STYLE);
+                if resizable {
+                    style |= (WS_THICKFRAME | WS_MAXIMIZEBOX) as LONG_PTR;
+                } else {
+                    style &= !(WS_THICKFRAME | WS_MAXIMIZEBOX) as LONG_PTR;
+                }
+                SetWindowLongPtrW(hwnd, GWL_STYLE, style);
```

Comment:
Need to check for errors here. `SetWindowLongPtrW` returns zero in case of error. As for what to do about it, should log the error I guess.